src/types.rs

```diff
@@ -665,6 +665,30 @@ impl<'a> TryFrom<&'a CertificateDer<'_>> for EncodedCertificateIdentifier {
     }
 }
 
+/// Information about a suggested renewal window for a certificate
+///
+/// See <https://www.ietf.org/archive/id/draft-ietf-acme-ari-07.html#section-4.2>
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RenewalInfo {
+    /// The suggested renewal window for a certificate
+    pub suggested_window: SuggestedWindow,
+    /// A URL to a page explaining why the suggested renewal window has its current value
+    pub explanation_url: Option<String>,
+}
+
+/// A suggested renewal window for a certificate
+///
+/// See <https://www.ietf.org/archive/id/draft-ietf-acme-ari-07.html#section-4.2>
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SuggestedWindow {
+    /// The RFC3339 formatted start date/time of the suggested renewal window
+    pub start: String,
+    /// The RFC3339 formatted end date/time of the suggested renewal window
+    pub end: String,
```

Comment:
I think these fields should be `DateTime<Utc>` directly. Maybe that means that `renewal_info()` is `cfg(feature = "chrono")`?

Reply:
Done.